src/cpp/server/auth/ServerAuthHandler.cpp

```diff
@@ -454,9 +453,17 @@ Error initialize()
       return error;
    }
 
+   core::system::User serverUser;
+   error = core::system::User::getUserFromIdentifier(options().serverUser(), serverUser);
+   if (error)
+   {
+      LOG_ERROR_MESSAGE("Could not get server user details");
```

Comment:
Feels like this should be metadata attached to the error rather than logged separately?

Reply:
(i.e.  we typically only `LOG_ERROR_MESSAGE` at the point when we abort an operation, not as part of bubbling an error through)